share/OpenMS/CHEMISTRY/XTandem_default_input.xml

```diff
@@ -76,7 +76,7 @@
   <note type="input" label="protein, cleavage semi">no</note>
   <note type="input" label="protein, quick acetyl">yes</note>
   <note type="input" label="protein, quick pyrolidone">yes</note>
-  <note type="input" label="protein, modified residue mass file"></note>
+  <note type="input" label="protein, modified residue mass file"></note>               !!!!! fill in a filename here for 15N labeling etc.. make sure the PATH is valid, or Tandem will silently ignore it!

```

Comment:
what is this?

Reply:
a comment :)
since its chars between tags, the parser will ignore it